tests/installation/bootloader.pm

```diff
@@ -24,17 +24,21 @@ use bootloader_spvm;
 use registration;
 use version_utils qw(:VERSION :SCENARIO);
 use utils;
-use Utils::Architectures qw(is_i586 is_i686);
 use Utils::Backends 'is_spvm';
 
 # hint: press shift-f10 trice for highest debug level
 sub run {
     return boot_spvm if is_spvm;
     return           if pre_bootmenu_setup == 3;
     return           if select_bootmenu_option == 3;
+    # the default loader is isolinux on openSUSE/SLE products with product-builder
     my $boot_cmd = 'ret';
     # Tumbleweed livecd has been switched to grub with kiwi 9.17.41 except 32bit
-    uefi_bootmenu_params() if (is_livecd && is_tumbleweed && !is_i586 && !is_i686);
+    # when LIVECD_LOADER has set grub2 then change the boot command for grub2
+    if (is_livecd && check_var('LIVECD_LOADER', 'grub2')) {
+        $boot_cmd = 'ctrl-x';
+    }
+    uefi_bootmenu_params() if (is_livecd && check_var('LIVECD_LOADER', 'grub2'));
```

Comment:
The if-condition seems to be exactly the same as the if block just above - why not take it into the if block?

Reply:
yeah, idk why I didn't do that. :) Thanks.